### ISSUE TITLE
Pressing "Edit" on a bot seems to bring you to a 403 error

### ISSUE THREAD

**Issue by muhimur9049** [top-gg/issues#612]
#### Expected Behavior

Editing my bot (description, icon, etc.)

#### Observed Behavior

This leads to a 403 forbidden error

#### Applicable Links or Attachments

![Error](https://i.imgur.com/vyqOn7I.png)

#### Steps to Reproduce

1. Open top.gg
2. Go to a bot you own
3. Click "Edit"
(Not sure if this happens for everyone, but for one of my bots it happens)

#### Web Browser and Version

Version 89.0.4389.90 (Official Build) (64-bit)

**Comment by Hopefuls** [comment]
This issue is currently known and the team is aware of it.

**Comment by Xignotic84** [comment]
Are you listed as a co-owner or do you own the bot?

**Comment by muhimur9049** [comment]
Well I mean I own the bot, yeah, but I'm listed as a co-owner on the top.gg bot page with 2 other people

I've always been able to edit the bot before though

**Comment by Xignotic84** [comment]
This is because we've removed the ability for co-owners to edit bots unless the bot is managed by a team and the co-owners are within that team.

**Comment by muhimur9049** [comment]
So for me to fix it, I just need to create a Team for it?

**Comment by Xignotic84** [comment]
If you transfer your bot into a team and your team members have the necessary permissions within that team, they should able to edit the bot.

**Comment by Taucher2003** [comment]
> This is because we've removed the ability for co-owners to edit bots unless the bot is managed by a team and the co-owners are within that team.

Why did you removed that? It is a bit dumb, that you have to create a Team for just one bot.

**Comment by TheNoob27** [comment]
> I'm listed as a co-owner on the top.gg bot page with 1 other person

> This is because we've removed the ability for co-owners to edit bots unless the bot is managed by a team and the co-owners are within that team.

Wouldn't you just need to unlist yourself as a co-owner?

**Comment by muhimur9049** [comment]
I’m honestly confused now because when I brought this up to Veld, he said he’d look into it 

But when I brought it up on the GitHub, I am first told it’s an issue that is known and that the Team is aware of it

Later on I get told it’s because they removed the ability for co-owners to edit bots...? If that’s the case, that’s a little dumb because what’s the point of that? Shouldn’t co-owners of the bot be able to edit bots without the need of having to create a team for just one bot?

**Comment by ChillFish8** [comment]
I agree with @muhimur9049 on this if what @Xignotic84 is correct, the whole point of someone being a co-owner is because they co-own the bot, and they should be trusted enough to be able to edit the bot without neededing to be in a team. Just seems like a pointless extra set of steps and pointless extra setup for things to go wrong if you ask me for no gain.

**Comment by muhimur9049** [comment]
I can now edit the bot again without a 403 forbidden error...

Did you guys fix the issue for everyone or did it just fix on its own for me lol?
